test/contrib/gtp.uts

```diff
@@ -16,6 +16,25 @@ assert a.E == a.S == a.PN == 0
 a = GTPHeader(raw(GTP_U_Header()/GTPErrorIndication()))
 assert isinstance(a, GTP_U_Header)
 
+a.show(dump=True)
```

Comment:
Could you use the same kind of tests done later in the file?
- dissection: https://github.com/secdev/scapy/blob/master/test/contrib/gtp.uts#L57
- instanciation: https://github.com/secdev/scapy/blob/master/test/contrib/gtp.uts#L73

Reply:
I modified the test for gtp pdu session container. I removed show and hexdump to check the header. Instead, I added some assert to check if the header is correct or not.